.github/workflows/release_synapse.yml

```
@@ -0,0 +1,66 @@
+          - PROD CENTRAL
+
+jobs:
+  publish-release:
+    runs-on: ubuntu-22.04
+    if: github.repository_owner == 'wso2'
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 21
+        uses: actions/setup-java@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)